test/integration/topical_event_about_page_test.rb

```diff
@@ -49,9 +49,54 @@ class TopicalEventAboutPageTest < ActionDispatch::IntegrationTest
     refute page.has_css?('.contents-list.contents-list-dashed')
   end
 
+  test "contents list not displayed when fewer than three items" do
+    @content_item = get_content_example("topical_event_about_page")
+    @content_item["details"]["body"] = body_with_two_contents_list_items
+
+    content_store_has_item(@content_item["base_path"], @content_item.to_json)
+
+    visit @content_item["base_path"]
+    refute page.has_css?(".app-c-contents-list")
+  end
+
+  test "contents list displayed when fewer than three items and first item word count is greater than 100" do
+    @content_item = get_content_example("topical_event_about_page")
+    @content_item["details"]["body"] = long_first_item_body
+
+    content_store_has_item(@content_item["base_path"], @content_item.to_json)
+
+    visit @content_item["base_path"]
+    assert page.has_css?(".app-c-contents-list")
+  end
+
 private
 
   def topical_event_end_date
     Date.parse(@content_item['links']['parent'][0]['details']['end_date'])
   end
+
+  def long_first_item_body
+    "<div class='govspeak'><h2 id='response-in-the-uk'>Item 1</h2>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean facilisis
```

Comment:
The [Faker gem](https://github.com/stympy/faker/blob/master/doc/lorem.md) is great for generating this sort of thing.

Reply:
Nice one!  Will keep that in mind for production tests 👍

Reply:
We can also use the govuk_schema gem to generate random examples to test this with